apps/web/src/pages/integrations/components/Modal/ConnectIntegrationForm.tsx

```diff
@@ -196,7 +198,10 @@ export function ConnectIntegrationForm({
           data: { credentials, active: isActive, check },
         });
       }
-      await refetch();
+      await queryClient.refetchQueries({
+        predicate: ({ queryKey }) =>
+          queryKey.includes(QueryKeys.integrationsList) || queryKey.includes(QueryKeys.activeNotificationsList),
+      });
```

Comment:
on successful integration settings form submit, refetch the active and all integrations